docs/AddingNewProjects.md

```diff
@@ -0,0 +1,51 @@
+# Adding New Projects To The Repo
+
+Sample PR of final result: https://github.com/dotnet/aspnetcore/pull/41945
+
+## Creating a new project
+1. Create a new folder that will house your `.csproj` and other project-related files.
+2. (EXTREMELY IMPORTANT) Inside our new folder make a new folder called `src`.
```

Comment:
I still suggest making it clear `src/` is just one option. When adding a `test/` project or similar the issues in (4) do not apply.